a03993/MERN-project
# feat: Add authentication route and integrate with main server

## Description


This PR introduces a new authentication route under the `feature/auth-route` branch. It includes the following changes:

- Added an authentication router module (`auth-route.js`):

  - Imported `express` and created a new router instance.
  - Implemented a middleware to log incoming requests at the authentication route.
  - Created a test endpoint (`GET /testAPI`) to verify the connectivity and response from the authentication route.

These changes are intended to set up a basic structure for the authentication routes, which can be expanded in future iterations.




## Type of Change



- [ ] Documentation update (adds or updates documentation)

## Checklist



## Related Issues


None

## Testing


1. Start the server and navigate to the `/testAPI` route (e.g., `http://localhost:3000/testAPI`).
2. Confirm that the server logs the message `Is receiving a request at auth route`.
3. Verify that the response message `Successfully connected to auth route...` is returned.
